packages/account/src/Components/poi/status/unverified/__test__/unverified.spec.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { PlatformContext } from '@deriv/shared';
+import { Unverified } from '../unverified.jsx';
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Icon: jest.fn(() => <div data-testid='dt_mocked_icon' />),
+    };
+});
+
+describe('Unverified', () => {
+    const renderWithRouter = component =>
+        render(<PlatformContext.Provider value={{ is_appstore: true }}>{component}</PlatformContext.Provider>);
+
+    it('should render unverified component', () => {
+        renderWithRouter(<Unverified />);
+
+        expect(screen.getByText(/We could not verify your proof of identity/i)).toBeInTheDocument();
+        expect(screen.getByTestId(/dt_mocked_icon/)).toBeInTheDocument();
+    });
+
+    it('should show description message', () => {
+        renderWithRouter(<Unverified is_description_enabled={true} />);
+
+        expect(
+            screen.getByText(
+                /As a precaution, we have disabled trading, deposits and withdrawals for this account. If you have any questions, please go to our Help Center/i
+            )
```

Comment:
We can test- On clicking help-center text, whether the page redirects to help centre page